### DISCUSSION TITLE
Quit to project list shortcut is very hard to press on mac

### DISCUSSION THREAD

**Discussion by passivestar** [godotengine/godot-proposals#9569] [Editor]
I click on the menu item instead because I'm not sure how to bend my fingers for 3 modifier keys in one row and Q. Am I the only one?

<img width="327" alt="image" src="https://github.com/godotengine/godot-proposals/assets/60579014/f01bb023-6081-4a89-bb49-67fd0c8f6f96"><br>

<img width="327" alt="image" src="https://github.com/godotengine/godot-proposals/assets/60579014/6c1a53ca-272e-4e68-abe5-90de2c21ce02"><br>

Can we maybe make it `Alt+Shift+Q` instead?

**Comment by aXu-AP** [comment]
I don't use mac so I don't know if there's some conventions or other reasons which contributed to that shortcut, but that definitely seems painful! For the reference, on Windows it's `Ctrl+Shift+Q`, so wouldn't mac equivalent be `Cmd+Shift+Q`?

**Reply by passivestar** [reply]
it's taken by an OS shortcut

**Comment by bruvzg** [comment]
> Can we maybe make it Alt+Shift+Q instead?

Some macOS keyboard layouts use `⌥ Opt + Letter` to input extra characters (e.g, `⌥ Opt + ⇧ Shift + Q` is `€` character in Latvian layout, some other similar combinations are used for normal letters, so it's not just a shortcut for special symbols), so it can't be used as default (you should be able to rebind it in the editor settings). This shortcut (and some other) is hard to use, but it's because we run out of usable combinations.

**Reply by bruvzg** [reply]
For the reference: to remap it, it's `editor/quit_to_project_list` in the editor input map.
